src/OrbitQt/orbitmainwindow.h

```diff
@@ -163,6 +164,9 @@ class OrbitMainWindow : public QMainWindow {
 
   static const QString kCollectThreadStatesSettingKey;
   void LoadCaptureOptionsIntoApp();
+  bool ShowCloseWarningAndReturnResult();
+  bool ShowCaptureAbortWarningAndReturnResult();
+  bool ExitRequestWithReturnCode(int return_code);
```

Comment:
It took me a while to understand the function name. I was always thinking: "What kind of request is exited here?".

Have you considered calling it `RequestExitWithReturnCode`. I think this is not ambiguous, but I might be wrong.

Reply:
Good idea! I renamed it

Reply:
Good catch. I compared to nullptr in both ifs.